### PR TITLE
Feature/refactoring footer update

### PR DESCRIPTION
**Pull request by CathyJinJNCC**

### CODE REVIEW

File: Catalogue.Web/app/js/controllers/OpenDataPublishingController.coffee [hunk 1]
```diff
@@ -1,34 +1,76 @@
...
         m =
-            tab: 2 # default to second UI tab
+            tab: 5 # default to fifth UI tab
             openData:
                 summary: {}
                 list: []
         
         $scope.m = m
+        $scope.signOffStatus = {}
+        $scope.recordTimeoutMap = {}
```
Comment (sumothecat) [inline Catalogue.Web/app/js/controllers/OpenDataPublishingController.coffee]: I would rename the `recordTimeoutMap` to `signOffTimeoutMap` or similar - both these new $scope props are specifically to do with the sign off tab.

File: Catalogue.Web/app/views/opendatapublishing/signoff.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        <thead>
+        <tr>
+            <th class="first"></th>
+            <th>Metadata date</th>
+            <th>Gemini</th>
+            <th ng-if="m.tab != 5">Signed Off</th>
+            <th ng-if="m.tab == 5">Assessed by</th>
+            <th ng-if="m.tab != 5">Last attempt</th>
+            <th ng-if="m.tab != 5">Last success</th>
+            <th ng-if="m.tab != 5"></th>
```
Comment (sumothecat) [inline Catalogue.Web/app/views/opendatapublishing/signoff.html]: Would specifying a width for this Sign Off column stop the UI jumping as the button is pressed?

**Comment by CathyJinJNCC** [comment]
Thanks for the comments :) Will go through them

File: Catalogue.Web/app/index.html [hunk 3]
```diff
@@ -75,6 +75,13 @@ <h1 title="Topcat">
                     </a>
                 </h1>
             </div>
+            <div class="pending-sign-off"
+                 ng-if="recordsPendingSignOff">
+                <a href="#opendatapublishing/signoff" class="btn btn-danger">
```
Comment (CathyJinJNCC) [inline Catalogue.Web/app/index.html]: You're right, I got carried away changing all the siros to sign offs and didn't notice this

File: Catalogue.Web/app/views/opendatapublishing/signoff.html (hunk 2 above)
Comment (CathyJinJNCC) [inline Catalogue.Web/app/views/opendatapublishing/signoff.html]: Ah does it do that for you? Will investigate

File: Catalogue.Web/app/js/config.coffee [hunk 4]
```diff
@@ -0,0 +1,3 @@
+﻿module = angular.module 'app.config'
+
+module.constant 'signOffGroup', 'OpenDataIAO'
```
Comment (CathyJinJNCC) [inline Catalogue.Web/app/js/config.coffee]: Forgot to point out I added this config module to put the group name in. Wasn't really sure how to share these kind of global variables, apparently this is a common way but it means we'll need to update the group name in two places if it changes.

File: Catalogue.Web/app/js/controllers/MainController.coffee [hunk 5]
```diff
@@ -28,4 +28,10 @@
         # horrid hack to ensure qtips hide when url (location) changes
         # (tooltips left hanging visible when the element has gone)
         $rootScope.$on '$locationChangeStart', -> $('.qtip').qtip 'hide'
+
+        # global notification for records pending sign off
+        $scope.recordsPendingSignOff = false
+        $scope.loadRecordsPendingSignOff = -> $http.get('../api/publishing/opendata/pendingsignoff').success (result) -> $scope.recordsPendingSignOff = result.length > 0
```
Comment (CathyJinJNCC) [inline Catalogue.Web/app/js/controllers/MainController.coffee]: There's the argument that an assessor will want to confirm that their record is actually pending sign off and that's currently the only way to see that, probably up for more discussion?

**Pull request merged**